notifications_utils/sanitise_text.py

```diff
@@ -100,6 +100,7 @@ class SanitiseSMS(SanitiseText):
 
     # Welsh characters not already included in GSM
     WELSH_NON_GSM_CHARACTERS = set("ÂâÊêÎîÔôÛûŴŵŶŷ")
+    FRENCH_NON_GSM_CHARACTESR = set("ÀÂâçÈÊêËëÎîÏïÔôŒœÙÛû₣")
```

Comment:
+1 on building a different set instead of complementing the existing non GSM characters, this makes it much clearer and this is the right data structure to use to avoid duplicates